src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -29,12 +29,101 @@
 import com.puppycrawl.tools.checkstyle.api.TokenTypes;
 
 /**
- * Checks the npath complexity against a specified limit (default = 200).
```

Comment:
I think the description is better with this as the first sentence, minus the default.

Reply:
done.